apps/server/src/services/RestoreService.ts

```diff
@@ -0,0 +1,106 @@
+import { Playback } from 'ontime-types';
+import { logger } from '../classes/Logger.js';
+import { PlaybackService } from './PlaybackService.js';
+import { eventTimer } from './TimerService.js';
+import { EventLoader } from '../classes/event-loader/EventLoader.js';
+
+//File stuff
+import fs from 'fs';
+import { resolveDbDirectory } from '../setup.js'
+import path from 'path';
+import { Writer } from 'steno';
+
+type OntimeState = {
+    playback: Playback;
+    selectedEventId: string | null;
+    startedAt: number | null;
+    addedTime: number | null;
+    pausedAt: number | null
+}
+
+/**
+ * Service manages saveing of timer state
+ * that can then be resored when reopening
+ */
+export class RestoreService {
+    private lastStore: string = '';
+    private readonly file;
+    private load: Partial<OntimeState> = { playback: Playback.Armed };
+    private readonly ok;
+    private readonly filePath;
+
+    constructor(filePath: string) {
+        this.filePath = filePath;
+        try {
+            const data = fs.readFileSync(path.join(filePath, 'RestoreService.csv'), 'utf-8');
+            const elements = data.split(',');
+            if (elements[5] == '\n') {
+                logger.info('RESTORE', 'intact restore file');
+                this.load.playback = elements[0] as Playback;
+                this.load.selectedEventId = elements[1] != 'null' ? elements[1] : null;
+                this.load.startedAt = elements[2] != 'null' ? +elements[2] : null;
+                this.load.addedTime = elements[3] != 'null' ? +elements[3] : null;
+                this.load.pausedAt = elements[4] != 'null' ? +elements[4] : null;
```

Comment:
We likely can simplify here.
in short, going through the elements, we can cancel the process if any of them are undefined or do not match 

Try and use shorter syntax to make this easier to read, like (rough code written here, likely will not work)


```ts
const enumValues = Object.values(Playback) <--- we can make an utility 
function to give us the values of the enum

const maybePlayback = elements?.[0]
const isValidPlayback = maybePlayback && enumValues.includes(maybePlayback)

if (!isValidPlayback) { return } <--- short circuit if something is not right

```